docker/opbeans/frontend_nginx/entrypoint.sh

```diff
@@ -1,9 +1,10 @@
 #!/usr/bin/env bash
 echo "******************************************************************************************************************************************"
-echo "* You must define ELASTIC_OPBEANS_API_SERVER to reditect all request to http://host:port/api/* (default http://localhost:3000) *"
-echo "* You can define ELASTIC_APM_JS_BASE_SERVER_URL to send the APM request (default http://localhost:8200)                                         *"
+echo "* You must define ELASTIC_OPBEANS_API_SERVER to reditect all request to http://host:port/api/* (default http://localhost:3000)           *"
```

Comment:
Typo: `reditect > redirect`
```suggestion
echo "* You must define ELASTIC_OPBEANS_API_SERVER to redirect all request to http://host:port/api/* (default http://localhost:3000)           *"
```